wgengine/filter/filter_test.go

```diff
@@ -560,3 +570,6 @@ func netports(netPorts ...string) (ret []NetPortRange) {
 	}
 	return ret
 }
+
+func TestMatchesFromFilterRules(t *testing.T) {
```

Comment:
oops?

Reply:
oh, yes, I started to add this then realized we already had tests for it.

Reply:
done.